### DISCUSSION TITLE
Generate a new key pair ?

### DISCUSSION THREAD

**Discussion by korniand** [DanSheps/netbox-secretstore#88] [General]
Hi all,
When "Secrets" object was part of Netbox v2 core functionality, and you wanted to generate a user key for the first time, there was the blue button "Generate a new key pair", and then you could see a pop up window with your new private and public keys.
Since migration to Netbox v3 and installing secretstore plugin it seems it is no longer possible to generate a user key.
If I go to "Plugins" -> "Netbox SecretStore" -> "User Key" -> "Create a User Key" I can't find the button to generate new rsa key.
It is not available in admin section either.
Am I looking at the wrong place ? or we now have to generate an RSA key pair outside of Netbox gui first  and then just copy/paste the public key ?
Thanks

**Comment by greinick** [comment]
Hi, I have the same problem running on netbox 3.2.8

Regards . Götz

**Comment by DanSheps** [comment]
It looks like the views were re-written and there was a breaking change.  There will be a new release when version 3.3.0 is released that will resolve this.

**Reply by greinick** [reply]
Thanks!

**Reply by greinick** [reply]
I did the 3.3.0 Update and it works. Thanks!!
